src/libraries/System.Text.Json/tests/Serialization/ReadValueTests.cs

```diff
@@ -761,21 +761,6 @@ static void ReadAndValidate(byte[] utf8, JsonSerializerOptions options, JsonRead
         }
     }
 
-    public class FullNameNullTest
```

Comment:
Moved this test (deserializing a `typeof(List<>)`); it is now part of the added test below. We now throw an `NSE` upstream.